infra/build/functions/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
```

Comment:
Can you put this somewhere more specific? e.g. `infra/build/functions/ci_trial_build` instead of this directory which is overly broad.